client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -55,8 +53,6 @@ const ConnectedThemesSelection = connectOptions( ( props ) => {
 
 const ConnectedSingleSiteJetpack = connectOptions( ( props ) => {
 	const {
-		analyticsPath,
```

Comment:
Seems like we're still passing those `analyticsPath` and `analyticsPageTitle` props from `SingleSiteThemeShowcaseWithOptions`, but they're no longer necessary. Worth cleaning them up, even if not in this PR.

Reply:
They are passed through to `<ThemeShowcase` and used there for `<PageViewTracker>`.

Reply:
Yep. But they get passed along with the rest of the props down here as well.

Reply:
I'm afraid I don't understand. The `analytics*` props are passed through from the controller to `SingleSiteJetpack`, to `ThemeShowcase` and are finally used in `PageViewTracker`. We cannot remove them, because they are used. Or do you propose to clean up something else?

Reply:
Ah, I see what you mean. Let's leave them as they are for now then 👍  Having to pass them levels down like this definitely isn't great, but not such a pain point at this time.